src/mocking/data/raw/categories.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
```

Comment:
`All` shouldn't be a category by itself. It should only exist as part of the category picker